scipy/signal/_delegators.py

```diff
@@ -342,7 +344,7 @@ def firwin2_signature(numtaps, freq, gain, *args, **kwds):
     return array_namespace(freq, gain)
 
 
-def freqs_zpk_signature(z, p, k, worN, *args, **kwds):
+def freqs_zpk_signature(z, p, k, worN=200, *args, **kwds):
```

Comment:
Note that I can mutate the function signature and the tests still pass.

```diff
-def freqs_zpk_signature(z, p, k, worN=200, *args, **kwds):
+def freqs_zpk_signature(z, p, k, worN=200, **kwds):
     return array_namespace(z, p, worN)
```

That said, almost all other mutations cause failures, so pretty good overall.